src/routes/Profile/helpers.ts

```diff
@@ -314,7 +314,8 @@ export const getCommunitySbt = (signer: Signer | null) => {
   const params: SBTConstructorArgs = {
     id: process.env.REACT_APP_COMMUNITY_SBT_ADDRESS || '',
     signer: signer,
-    badgesSubgraphUrl: process.env.REACT_APP_SUBGRAPH_BADGES_URL,
+    currentBadgesSubgraphUrl: process.env.REACT_APP_CURRENT_SUBGRAPH_BADGES_URL,
```

Comment:
I would rather if we have a better configuration for this. For example: 

`badgesSubgraphUrls: Record<Season['id'], string>` so you define them as following:

```
0: process.env.REACT_APP_SOG_SUBGRAPH_BADGES_URL,
1: process.env.REACT_APP_SO1_SUBGRAPH_BADGES_URL,
2: process.env.REACT_APP_SO2_SUBGRAPH_BADGES_URL,
```

WDYT?

Reply:
we are only interested in current and next. the switch will happen automatically so I don't see the point in storing all of the subgraph urls